requirements.txt

```diff
@@ -17,7 +17,7 @@ sklearn
 seqeval
 mlflow==1.0.0
 # huggingface repository
-transformers==3.1.0
+transformers==3.3.1
```

Comment:
I think better to have like this, but I am not sure about implication on FARM -
```
transformers>=3.3.1
```

Reply:
I think it is bad idea to not use fixed version, after seeing these breaking changes :) 
https://github.com/huggingface/transformers/pull/1089/files

Reply:
Yes, pinned version is better as there are still many breaking changes in transformers